test/clj/game/cards/ice_test.clj

```diff
@@ -6266,7 +6266,7 @@
       (click-prompt state :runner "0")
       (is (= 1 (count-tags state)) "Trace succeeds with 1 advancement"))))
 
-(deftest searchlight-x-fn
+(deftest ^:kaocha/pending searchlight-x-fn
```

Comment:
Why is this one now broken?

Reply:
i ran the code and i don't see any reason why this would fail, so i'm confused what's happening. i'd prefer that the underlying issue is fixed and not merely disabling the test.

Reply:
oh, the test itself calls `disable-card` directly (the old bad one), it's on my list of things to fix

Reply:
oh i see. yeah, with that changed, it'll be good to go.

Reply:
ok, refactored it to use hush